starter_code/bin/seeds.js

```diff
@@ -0,0 +1,101 @@
+        showtimes: ["13:40", "15:50", "19:00", "21:20", "23:50"]
+    },
+    {
+        title: "Thoroughbreds",
+        director: "Cory Finley",
+        stars: ["Anya Taylor-Joy", "Olivia Cooke", "Anton Yelchin"],
+        image:
+            "https://images-na.ssl-images-amazon.com/images/M/MV5BNDcyNDA4NDAzN15BMl5BanBnXkFtZTgwODQxMDQ5NDM@._V1_UX182_CR0,0,182,268_AL_.jpg",
+        description:
+            "Two upper-class teenage girls in suburban Connecticut rekindle their unlikely friendship after years of growing apart. Together, they hatch a plan to solve both of their problems-no matter what the cost.",
+        showtimes: ["13:00", "15:30", "18:00", "20:10", "22:40"]
+    },
+    {
+        title: "The Leisure Seeker",
+        director: "Paolo Virzì",
+        stars: ["Helen Mirren", "Donald Sutherland", "Janel Moloney"],
+        image:
+            "https://images-na.ssl-images-amazon.com/images/M/MV5BMTg1NTg2MzcyNF5BMl5BanBnXkFtZTgwNjMwMDIzNDM@._V1_UX182_CR0,0,182,268_AL_.jpg",
+        description:
+            "A runaway couple goes on an unforgettable journey in the faithful old RV they call The Leisure Seeker, traveling from Boston to The Ernest Hemingway Home in Key West. They recapture their passion for life and their love for each other on a road trip that provides revelation and surprise right up to the very end.",
+        showtimes: ["13:40", "15:50", "19:00", "21:20", "23:50"]
+    },
+    {
+        title: "Black Panther",
+        director: "Ryan Coogler",
+        stars: ["Chadwick Boseman", "Michael B. Jordan", "Lupita Nyong'o"],
+        image:
+            "https://images-na.ssl-images-amazon.com/images/M/MV5BMTg1MTY2MjYzNV5BMl5BanBnXkFtZTgwMTc4NTMwNDI@._V1_UX182_CR0,0,182,268_AL_.jpg",
+        description:
+            "T'Challa, the King of Wakanda, rises to the throne in the isolated, technologically advanced African nation, but his claim is challenged by a vengeful outsider who was a childhood victim of T'Challa's father's mistake.",
+        showtimes: ["13:50", "16:20", "19:20", "22:10"]
+    },
+    {
+        title: "Red Sparrow",
+        director: "Francis Lawrence",
+        stars: ["Jennifer Lawrence", "Joel Edgerton", "Matthias Schoenaerts"],
+        image:
+            "https://images-na.ssl-images-amazon.com/images/M/MV5BMTA3MDkxOTc4NDdeQTJeQWpwZ15BbWU4MDAxNzgyNTQz._V1_UX182_CR0,0,182,268_AL_.jpg",
+        description:
+            "Ballerina Dominika Egorova is recruited to 'Sparrow School,' a Russian intelligence service where she is forced to use her body as a weapon. Her first mission, targeting a C.I.A. agent, threatens to unravel the security of both nations.",
+        showtimes: ["13:00", "15:30", "18:00", "20:10", "22:40"]
+    }
+];
+
+Movie.create(movies, err => {
```

Comment:
from line 5: mongoose.connect(`mongodb://localhost/${dbName}`, {

Mongoose.connect I'm pretty sure this is an async function, and thus it will be more solid if you put this Movie.create into a .then function after the mongoos.connect just to make sure that you are connected to the DB before you add the Movies